spec/forms/placements/placements/filter_form_spec.rb

```diff
@@ -110,19 +110,19 @@
 
     context "when removing partner school id params" do
       let(:params) do
-        { partner_school_ids: %w[school_id_1 school_id_2] }
+        { only_partner_schools: true }
       end
 
       it "returns the placements index page path without the given school id param" do
         expect(
           filter_form.index_path_without_filter(
             provider:,
-            filter: "partner_school_ids",
-            value: "school_id_1",
+            filter: "only_partner_schools",
+            value: false,
           ),
         ).to eq(
           placements_provider_placements_path(provider, filters: {
-            partner_school_ids: %w[school_id_2],
+            partner_schools: [],
```

Comment:
Shouldn't this be:

```suggestion
            only_partner_schools: false,
```

? 🤔

I think we're getting a false-positive because `partner_schools: []` is no longer a defined filter.